src/main/java/minicraft/screen/entry/StringEntry.java

```diff
@@ -56,5 +58,5 @@ public void tick(InputHandler input) {}
 	public int getColor(boolean isSelected) { return color; }
 
 	@Override
-	public String toString() { return text; }
+	public String toString() { return localize? Localization.getLocalized(text): text; }
 }
```

Comment:
Why did you do this. This does nothing other than to increase CPU usage.

Reply:
Well, some StringEntry might still use the past localized string instead of the new one.

Reply:
We will do that if it is needed. There isn't even a function to access it. Revert this for now.

Reply:
The TitleDisplay version check is not localized at start.

Reply:
And how does that have anything to do with this?

Reply:
it is because the localization is not ready. the #render method could always refresh this, even the localization is changed.

Reply:
Localization should probably be loaded before it then.

Reply:
Localization is packed with resource packs.